app/api/Files.scala

```diff
@@ -716,6 +716,13 @@ class Files @Inject()(
   }
 
   def jsonFile(file: File, serverAdmin: Boolean = false): JsValue = {
+    val foldersContainingFile = folders.findByFileId(file.id)
+    val allPaths: List[List[String]] =  (for (folder <- foldersContainingFile) yield (folderPath(folder, List()).tail))
+    var path_str = allPaths.map(xl => "/" + xl.map(x => x.toString()).mkString("/")).mkString("")
+    if(path_str == "")
```

Comment:
can you add the `{` on the same line as the if

Reply:
done as: if(path_str == "") {